packages/dev/core/src/Rendering/boundingBoxRenderer.ts

```diff
@@ -162,6 +178,29 @@ export class BoundingBoxRenderer implements ISceneComponent {
     private _uniformBufferFront: UniformBuffer;
     private _uniformBufferBack: UniformBuffer;
     private _renderPassIdForOcclusionQuery: number;
+    /**
+     * Internal buffer for instanced rendering
+     * @internal
+     */
+    private _matrixBuffer: Nullable<Buffer> = null;
+    private _matrices: Nullable<Float32Array> = null;
+
+    /**
+     * Internal state of whether instanced rendering enabled
+     * @internal
+     */
+    protected _useInstances = false;
+    /**
+     * Choose whether dynamic vertex buffer should be used for instanced rendering.
+     * If true, the buffer would be updated instead of recreated when rendering a frame,
+     * this could reduce memory pressure.
+     */
+    public updatable = true;
+
+    /** @internal */
+    public _drawWrapperFront: Nullable<DrawWrapper> = null;
+    /** @internal */
+    public _drawWrapperBack: Nullable<DrawWrapper> = null;

```

Comment:
```suggestion
    private _drawWrapperFront: Nullable<DrawWrapper> = null;
    private _drawWrapperBack: Nullable<DrawWrapper> = null;
```
I don't think we need to access these variables outside the class?

Reply:
If changed to `private`, type check in `_createWrappersForBoundingBox` would not pass, see CI logs [here](https://dev.azure.com/babylonjs/ContinousIntegration/_build/results?buildId=35461&view=logs&jobId=8c3f0989-617d-5669-63ad-dc4309b25f44&j=8c3f0989-617d-5669-63ad-dc4309b25f44&t=1f7a1b4b-d9c8-5792-a01a-a4c2a11af4fd).
```
src/Rendering/boundingBoxRenderer.ts(451,26): error TS2339: Property '_drawWrapperFront' does not exist on type 'BoundingBox | BoundingBoxRenderer'.
src/Rendering/boundingBoxRenderer.ts(454,25): error TS2339: Property '_drawWrapperFront' does not exist on type 'BoundingBox | BoundingBoxRenderer'.
src/Rendering/boundingBoxRenderer.ts(455,25): error TS2339: Property '_drawWrapperBack' does not exist on type 'BoundingBox | BoundingBoxRenderer'.
src/Rendering/boundingBoxRenderer.ts(457,25): error TS2339: Property '_drawWrapperFront' does not exist on type 'BoundingBox | BoundingBoxRenderer'.
src/Rendering/boundingBoxRenderer.ts(458,25): error TS2339: Property '_drawWrapperBack' does not exist on type 'BoundingBox | BoundingBoxRenderer'.
```